modules/ROOT/pages/rtc-getting-started.adoc

```diff
@@ -92,3 +92,14 @@ tinymce.init({
 ----
 
 For details on the available configuration options, see: xref:rtc-options-overview.adoc[Configuration Options].
+
+An alternative to this documentation is a series of https://youtube.com/watch?v=zvu-7w-94vI&list=PLE6i0Bfo_RjlVyzACyh7XnnXJLoXD7h20[8 videos on setting up Real-Time Collaboration with TinyMCE]:
```

Comment:
```suggestion
An alternative to this documentation is a series of https://www.youtube.com/playlist?list=PLE6i0Bfo_RjlVyzACyh7XnnXJLoXD7h20[8 videos on setting up Real-Time Collaboration with TinyMCE]:
```
This might be a better link? As the current link points to the second video which is rather confusing.

Reply:
> As the current link points to the second video which is rather confusing.

I know I checked that at least once before committing. Obviously checking it twice was what I should have done. Thanks for catching that.